.github/workflows/ci_push_testing.yml

```diff
@@ -384,3 +384,29 @@ jobs:
         GANGA_GITHUB_HOST: True
         MONGODB_HOST: mongodb
         MONGODB_PORT: ${{ job.services.mongodb.ports[27017] }}
+'''
+  condor-unit:
+    name: Condor tests
+    needs: lint
+    runs-on: ubuntu-latest
+    container: centos:7
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python 3
+        run: yum install -y python3
+      - name: Install git
+        run: yum install -y git
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip setuptools wheel
+          python3 -m pip install -e .[dev]
+      -name: Install Condor
+        run: |
+          yum install -y condor
+          yum condor_master
```

Comment:
This doesn't look valid.

Reply:
I will look into this, the sudo commands in this order work on ubuntu but i am unaware as of now how the work on CentOS